### ISSUE TITLE
Defining milestones required for release stable version.

### ISSUE THREAD

**Issue by pawaclawczyk** [ClearcodeHQ/command-bus-console#18]
Hello everyone!

In order to release stable (v1.0.0) version I would like to discuss what features should be include.
In my opinion we should provide those three features:
- Support for Symfony 3.0 or at least get rid of deprecations in Symfony 2.8. This step also include providing this feature to [Symfony Console Form](https://github.com/matthiasnoback/symfony-console-form). See: [PR #11](https://github.com/matthiasnoback/symfony-console-form/pull/11)
- Automatically generate and register form types basing on command classes.
- Add support for more command buses, eg. [Tactician](https://github.com/thephpleague/tactician), [Broadway](https://github.com/qandidate-labs/broadway).

@krymen @Lucaszz @JacekJagiello waiting for your opinion ;)

**Comment by lzakrzewski** [comment]
Yes, We should change this component to more universal. If not, then no one will be using it.
- `+ 1` for support Tactician, Broadway,
- `+ 1` to require https://github.com/Lucaszz/SymfonyFormGenerator,
- `+ 1` to full 3.0 support.

**Comment by krymen** [comment]
I mostly agree, but automatical generation and registration of form types can be omitted before `v1.0.0`. If we introduce it later, it won't be a BC break as it is an additional feature on top of what we have now.

Imho first stable version should have only those features which introduce BC breaks to the existing codebase.
